packages/babel-plugin-jest-hoist/src/index.ts

```diff
@@ -6,8 +6,22 @@
  *
  */
 
-import type {NodePath, Visitor} from '@babel/traverse';
-import type {Identifier} from '@babel/types';
+import type {NodePath} from '@babel/traverse';
+import {
+  Expression,
+  Identifier,
+  Node,
+  Program,
+  callExpression,
+  identifier,
+  isIdentifier,
+} from '@babel/types';
+import {statement} from '@babel/template';
+import type {PluginObj} from '@babel/core';
+
+const JEST_GLOBAL_NAME = 'jest';
+const JEST_GLOBALS_MODULE_NAME = '@jest/globals';
+const JEST_GLOBALS_MODULE_JEST_EXPORT_NAME = 'jest';
```

Comment:
any reason to have both this and the one on line 22?

Reply:
One is for the name of the global we inject into the test env, one is for the name of the `@jest/globals` export. I wanna be clear about which part of the plugin deals with which of those two things.